Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -477,6 +482,11 @@ static void softlinkDataDetectorsFrameworks()
     // Soft link frameworks related to Data Detection before we disconnect from launchd because these frameworks connect to
     // launchd temporarily at link time to register XPC services. See rdar://93598951 (my feature request to stop doing that)
     softlinkDataDetectorsFrameworks();
+
+#if HAVE(VIDEO_RESTRICTED_DECODING) && PLATFORM(MAC)
+    if (codeCheckSemaphore)
+        dispatch_semaphore_wait(codeCheckSemaphore.get(), dispatch_time(DISPATCH_TIME_NOW, 50 * NSEC_PER_MSEC));
```

Comment:
Don't we need to wait forever here? Otherwise, in outlier cases where the trustd check-in takes more than 50ms, we'll get a process with mystery media failures.

Reply:
> Don't we need to wait forever here? Otherwise, in outlier cases where the trustd check-in takes more than 50ms, we'll get a process with mystery media failures.

Ah, that is a good point! Will fix.

Thanks for reviewing!